recipes/mammoth/fix_build_on_windows.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/setup.py b/setup.py
+index 0e4c48f..f3c6ce3 100644
+--- a/setup.py
++++ b/setup.py
+@@ -4,7 +4,7 @@ import os
+ from setuptools import setup
+
+ def read(fname):
+-    return open(os.path.join(os.path.dirname(__file__), fname)).read()
++    return open(os.path.join(os.path.dirname(__file__), fname), encoding="utf8").read()
```

Comment:
Not doing so yielded errors when building this library on windows.
```
  Traceback (most recent call last):
    File "D:\bld\mammoth_1734340231977\_h_env\lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 353, in <module>
      main()
    File "D:\bld\mammoth_1734340231977\_h_env\lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 335, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
    File "D:\bld\mammoth_1734340231977\_h_env\lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 149, in prepare_metadata_for_build_wheel
      return hook(metadata_directory, config_settings)
    File "D:\bld\mammoth_1734340231977\_h_env\lib\site-packages\setuptools\build_meta.py", line 366, in prepare_metadata_for_build_wheel
      self.run_setup()
    File "D:\bld\mammoth_1734340231977\_h_env\lib\site-packages\setuptools\build_meta.py", line 311, in run_setup
      exec(code, locals())
    File "<string>", line 14, in <module>
    File "<string>", line 7, in read
    File "D:\bld\mammoth_1734340231977\_h_env\lib\encodings\cp1252.py", line 23, in decode
      return codecs.charmap_decode(input,self.errors,decoding_table)[0]
  UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 3430: character maps to <undefined>
```
[relevant CI job](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1111141&view=logs&j=53c3d5f4-e7c7-5b53-4b94-211701b2ba17&t=8f042650-8479-5761-7a4a-8d021c0bb2de)